editor.planx.uk/src/routes/teamSettings.tsx

```diff
@@ -47,11 +47,11 @@ const teamSettingsRoutes = compose(
                 route: "design",
                 Component: DesignSettings,
               },
-              // {
-              //   name: "General",
-              //   route: "general",
-              //   Component: GeneralSettings,
-              // },
+              {
+                name: "General",
+                route: "general",
+                Component: GeneralSettings,
+              },
```

Comment:
nit: I think I'd reorder this array to be General → Design → Team

Reply:
@DafyddLlyr do you think it is worth doing these types of changes as a final PR when error handling is all sorted?

Reply:
Thought about it again, and tit doesn't really make it difference, so I've added it to this PR